smac/utils/sklearn_wrapper.py

```diff
@@ -0,0 +1,392 @@
+            method for sampling (such as those from scipy.stats.distributions).
+            If a list is given, it is sampled uniformly.
+
+        n_iter : int, default=10
+            Number of parameter settings that are sampled. n_iter trades
+            off runtime vs quality of the solution.
+
+        scoring : string, callable or None, default=None
+            A string (see model evaluation documentation) or
+            a scorer callable object / function with signature
+            ``scorer(estimator, X, y)``.
+            If ``None``, the ``score`` method of the estimator is used.
+
+        fit_params : dict, optional
+            Deprecated feature of Scikit-learn
+
+        iid : boolean, default=True
+            If True, the data is assumed to be identically distributed across
+            the folds, and the loss minimized is the total loss per sample,
+            and not the mean loss across the folds.
+
+        refit : bool
+            Deprecated feature of Scikit-learn
+
+        cv : int, cross-validation generator or an iterable, optional
+            Determines the cross-validation splitting strategy.
+            Possible inputs for cv are:
+              - None, to use the default 3-fold cross validation,
+              - integer, to specify the number of folds in a `(Stratified)KFold`,
+              - An object to be used as a cross-validation generator.
+              - An iterable yielding train, test splits.
+
+            For integer/None inputs, if the estimator is a classifier and ``y`` is
+            either binary or multiclass, :class:`StratifiedKFold` is used. In all
+            other cases, :class:`KFold` is used.
+
+            Refer :ref:`User Guide <cross_validation>` for the various
+            cross-validation strategies that can be used here.
+
+        refit : boolean, default=True
+            Refit the best estimator with the entire dataset.
+            If "False", it is impossible to make predictions using
+            this RandomizedSearchCV instance after fitting.
+
+        verbose : integer
+            Controls the verbosity: the higher, the more messages.
+
+        random_state : int or RandomState
+            Pseudo random number generator state used for random uniform sampling
+            from lists of possible values instead of scipy.stats distributions.
+
+        error_score : 'raise' (default) or numeric
+            Value to assign to the score if an error occurs in estimator fitting.
+            If set to 'raise', the error is raised. If a numeric value is given,
+            FitFailedWarning is raised. This parameter does not affect the refit
+            step, which will always raise the error.
+
+        return_train_score : boolean, default=True
+            If ``'False'``, the ``cv_results_`` attribute will not include training
+            scores.
+
+        intensification_percentage : float, default=None
```

Comment:
Hm, I do suggest a different way to pass arguments to SMAC, because the user might be interested in passing other ones, too. How about a dict mapping from argument to value?